From cd845304d95c54f218389e51b94c6b901b836727 Mon Sep 17 00:00:00 2001
From: Johan Hovold <johan@kernel.org>
Date: Mon, 13 Jul 2020 12:55:17 +0200
Subject: [PATCH 10/26] USB: serial: cp210x: use in-kernel types in port data

The port data is not exported to user space so use the in-kernel u8
type.

Link: https://lore.kernel.org/r/20200713105517.27796-6-johan@kernel.org
Reviewed-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Johan Hovold <johan@kernel.org>
---
 drivers/usb/serial/cp210x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/serial/cp210x.c b/drivers/usb/serial/cp210x.c
index 09445b7a8f649..d0c05aa8a0d6e 100644
--- a/drivers/usb/serial/cp210x.c
+++ b/drivers/usb/serial/cp210x.c
@@ -266,7 +266,7 @@ enum cp210x_event_state {
 };
 
 struct cp210x_port_private {
-	__u8			bInterfaceNumber;
+	u8			bInterfaceNumber;
 	bool			has_swapped_line_ctl;
 	bool			event_mode;
 	enum cp210x_event_state event_state;
-- 
2.32.0

